dev/package-examples/nginx-1.2.0/manifest.yml

```diff
@@ -12,3 +12,45 @@ requirement:
     versions: ">7.1.0 <7.6.0"
   elasticsearch:
     versions: ">7.0.1"
+
+datasources:
+  -
+    # Do we need a name for the data source?
+    name: nginx
+
+    # List of inputs this datasource supports
+    inputs:
+      -
+        # An id can be given, in case the type used here is not unique
+        # This is for selection in the stream
+        # id: nginx
+        type: metrics/nginx
+
+        # Common configuration options for this input
+        vars:
+          - name: hosts
+            description: Nginx hosts
+            default:
+              ["http://127.0.0.1"]
+            # All the config options that are required should be shown in the UI
+            required: true
```

Comment:
I wold like to see `type` defined explicitly for all vars. And if we like the `multi` suggestion in my previous comment, these should be added:
```yml
type: text
multi: true
```

Reply:
Agree. Adding.